### PR TITLE
User dropdown menu restyle and improvement.

### PR DESCRIPTION
**Pull request by bago2k4**
Card: https://trello.com/c/LaHWYKSn

Abstract: https://share.goabstract.com/bce92c7a-6c96-4c11-96f8-5753b89be0c4

To test:
- enter as Admin
- open the user menu
- [x] do you see all the needed menu items? Good
- [x] all the items work? Good
- enter as Author
- open the user menu
- [x] do you see all the needed menu items? Good
- [x] all the items work? Good
- enter as Viewer
- open the user menu
- [x] do you see all the needed menu items? Good
- [x] all the items work? Good

### CODE REVIEW

**Comment by bago2k4** [comment]
[UI Refresh: dropdown menu](https://trello.com/c/LaHWYKSn/1316-ui-refresh-dropdown-menu)

**Comment by thepug** [comment]
I don't see the new icons on my local dev. @bago2k4

**Comment by thepug** [comment]
After a refresh I see the icons.  However without a logo I see this:
screenshot

File: src/oc/web/actions/user.cljs [hunk 1]
```diff
@@ -378,6 +378,16 @@
 (defn user-profile-reset []
   (dis/dispatch! [:user-profile-reset]))
 
+(defn user-role []
```
Comment (thepug) [inline src/oc/web/actions/user.cljs]: This is user data related and belongs in the user store instead of action.

Comment (bago2k4) [reply src/oc/web/actions/user.cljs]: @thepug that's right but i can't access the user store directly from the UI or i'll break the pattern (check the graph, from the component i can only call an action creator, ie a user action).
WYT?

**Comment by bago2k4** [comment]
@thepug this is ready again

**Pull request merged**